javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/styles.css

```diff
@@ -1,128 +1,244 @@
+  --main-bg-color: #4d4d62;
+  --player-bg-color: #1b1b32;
+  --player-fg-color: #3b3b4f;
+  --player-highlight-color: #f1be32;
+}
+
 *,
-*::before,
-*::after {
-  margin: 0;
-  padding: 0;
+*::after,
+*::before {
   box-sizing: border-box;
 }
 
-:root {
-  --main-background-color: #0a0a23;
-  --gray100: #ffffff;
-  --gray200: #e9e8e8;
-  --golden-yellow: #feac32;
+body {
+  background-color: var(--main-bg-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
 }
 
-body {
-  background-color: var(--main-background-color);
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  row-gap: 5px;
 }
 
-.header {
+.player,
+.playlist {
+  width: 450px;
+  background-color: var(--player-bg-color);
+  border: 3px solid var(--player-fg-color);
+}
+
+.player {
+  height: 260px;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 10px;
+}
+
+.player__bar,
+.playlist__bar {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 25px;
+  justify-items: center;
+  align-items: center;
   text-align: center;
-  margin: 20px 0;
-  color: var(--gray100);
+  padding-left: 10px;
+  padding-right: 4px;
+  width: 430px;
+  height: 30px;
+  position: relative;
+  background-color: var(--player-fg-color);
 }
 
-.container {
-  width: 100%;
+.fcc-title {
+  font-size: 14px;
+  font-weight: 400;
 }
 
-.music-player {
+.fcc-title,
+.playlist-title {
+  color: var(--white-color, white);
+  margin: 0;
+  font-family: "Roboto Mono", monospace;
+}
+
+.player__content {
   display: flex;
-  background-color: var(--gray100);
-  width: 300px;
-  box-shadow: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
-  border-radius: 0.5rem;
+  background-color: var(--player-fg-color);
+  width: 430px;
+  height: 200px;
+  column-gap: 13px;
+  align-items: center;
+  justify-content: center;
+}
+
+#player__album-art {
+  width: 153px;
+  height: 153px;
   overflow: hidden;
-  margin: 0 auto;
+  background-color: var(--main-bg-color);
+  box-shadow: -6px 6px var(--player-bg-color);
+}
+
+#player__album-art img {
+  height: 153px;
+  width: 153px;
 }
 
-.flex-container {
+.player__display {
   display: flex;
-  width: 100%;
   flex-direction: column;
+  row-gap: 40px;
+  padding: 14px;
+  background-color: var(--player-bg-color);
+  height: 153px;
+  width: 226px;
 }
 
-.play-song-btn-container,
-.current-song-container {
+.player__display--songArtist > p {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis " [...]";
+}
+
+.player__buttons {
   display: flex;
-  padding: 5px;
-  border-bottom: 1px solid var(--main-background-color);
+  justify-content: space-around;
 }
 
-.song-btn {
-  padding: 5px;
-  font-size: 1rem;
-  background-color: var(--gray100);
+button {
+  background: transparent;
   border: none;
+  color: var(--white-color, white);
+  cursor: pointer;
+  font-size: 14px;
+  outline: none;
```

Comment:
Disabling the `outline` is considered an accessibility anti-pattern, especially if you don't replace it with `:focus-visible`. I understand that this course isn't about teaching accessibility, but I don't think we should encourage this as it is already far too commonly misused. Unless we want to get into the specifics of styling keyboard focus outlines I would suggest we remove this.

Reply:
Agreed. I'm leaving the outline and instead, I am adding some style to it.